### ISSUE TITLE
WSDL import - separator & in URL not decoded completely

### ISSUE THREAD

**Issue by starki7** [a85/POSTMan-Chrome-Extension#11329]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Description
In WSDL, URI is stored in **XML notation**, therefore **separator &** is stored as "**&amp;**".
When decoding URI, Postman recognizes &, but as parameter key it leaves "amp;".
It's easy to manually delete "amp;" from the parameter key, but it's annoying if you import WSDL's often.
### Example
_Part of WSDL with URL:_
```
<wsdl:service name="IsuMrAcmeHistoryService">
    <wsdl:port name="HTTP_Port" binding="p1:IsuMrAcmeHistoryBinding">
        <soap:address location="http://acme.org/SOAPAdapter/MessageServlet?senderParty=&amp;senderService=TIBCO&amp;receiverParty=&amp;receiverService=&amp;interface=IsuMrAcmeHistory&amp;interfaceNamespace=http%3A%2F%2Facme.org%2Finterfaces%2Facme%2Fisu%2Fmr%2Facme%2Fhistory%2Fv1"
            xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/"/>
    </wsdl:port>
</wsdl:service>
```

_Parsed URL_

```
{{HTTP_PortBaseUrl}}/SOAPAdapter/MessageServlet?senderParty=&amp;senderService=TIBCO&amp;receiverParty=&amp;receiverService=&amp;interface=IsuMracmeHistory&amp;interfaceNamespace=http%3A%2F%2Facme.org%2Finterfaces%2Facme%2Fisu%2Fmr%2Facme%2Fhistory%2Fv1
```

_Query Params Bulk Edit_
```senderParty:
amp;senderService:TIBCO
amp;receiverParty:
amp;receiverService:
amp;interface:IsuMrAcmeHistory
amp;interfaceNamespace:http%3A%2F%2Facme.org%2Finterfaces%2Facme%2Fisu%2Fmr%2Facme%2Fhistory%2Fv1
```

### Steps To Reproduce

1. Open workspace
2. Go to APIs
3. Go to Import
4. Import WSDL file
5. Go to the imported service
6. Check parameter key values
7. Check URL

### Operating System

Windows 10 Version 2004

### Postman Version

10.0.15

### Postman Platform

Postman App

**Comment by akshaydeo** [comment]
@starki7 thank you for reporting this issue. We are currently looking into it. I will update this ticket as soon as we have an update 🙇‍♂️

**Comment by VShingala** [comment]
@starki7 We've fixed this issue. Do try it out and let us know if you face any further issue here.

**Comment by starki7** [comment]
Hello!
I have tried it and I can confirm the fix.
I needed some time to get, that, it's not yet released to the postman web and postman desktop 10.7.0 apps.
Then, when I wanted to build it, I needed some time to find out which node version can be used to build it.
Are there any development instructions, other than postman-docs?
When can it be expected that the new versions 1.5.0 and 1.6.0 will be available in the desktop and web apps?
Thank You very much,
Sincerely Goran

**Comment by VShingala** [comment]
@starki7 So if you're a signed-in user then upon importing the definition, a collection get also imported within API. This flow will create a collection that has the fix already. Meanwhile offline flows require the latest version to be shipped that's v10.8 will be released next week or two.

For example, here's the behavior for signed in user that I've confirmed for mentioned issue that's fixed:

<img width="871" alt="image" src="https://user-images.githubusercontent.com/16756296/211311448-4f0dfb01-d820-47e6-9d66-5a13dc4e649d.png">

**Comment by starki7** [comment]
Hi!
Yes, now it's working in web and desktop app, many thanks,
Goran
